tests/test_halo.py

```diff
@@ -441,6 +442,25 @@ def test_animation_setter(self):
         spinner.animation = "marquee"
         self.assertEquals("marquee", spinner.animation)
 
+    def test_spinner_color(self):
+        """Test ANSI escape characters are present
+        """
+        colors = {
+            'cyan': Fore.CYAN,
+            'magenta': Fore.MAGENTA,
+            'yellow': Fore.YELLOW,
+            'blue': Fore.BLUE
+        }
+
+        for k, v in colors.items():
+            self._stream = io.open(self._stream_file, 'w+')  # reset stream
+            spinner = Halo(color=k, stream=self._stream)
```

Comment:
@eoinnoble This will become

```suggestion
            spinner = Halo(color=color_name, stream=self._stream)
```